### DISCUSSION TITLE
Add ability to rotate the token in any direction

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#12] [Ideas]
Maybe it is already possible? but haven't worked it out. If not, then perhaps add this ability via holding down LCtrl and roll mouse wheel to rotate the token clockwise.

This feature would be handy for combat, facing the enemy and also if there are sentry enemies placed on the map can be facing a specific direction which players or if playing solo you work which way the team or PC will approach a guarded building or gate etc.

**Comment by luiscastilho** [comment]
Not possible at the moment, but makes sense. I will add this idea to the TODO list as well.
